test/addon_data.cc

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#define NAPI_EXPERIMENTAL
+#include "napi.h"
+
+class Addon {
+ public:
+  bool verbose = true;
```

Comment:
Maybe we can try to store a reference of js value here too.

Reply:
Hmmm ... then the constructor should probably take a `Napi::Env` ...

Reply:
@legendecas storing a reference like this crashes because of #660. I can continue this once #662 lands.